index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/index.css">
+    <title>Game of Coffee</title>
+</head>
+<body>
+    <h1>Game of Coffee</h1>
+
+   <header>
+
+   </header>
+
+    <main>
+        <form id="user-sign-up">
+        <h2>Are you ready to play? Spoilers included.</h2>
+        <p>
+            <label for="name">Your Name: </label>
+            <input id="name" name="name" placeholder="type name here">
+        </p>
+
+        <fieldset>
+            <legend>
+                Choose Your House 
+            </legend>
+            <label>
+                <input type="radio" name="house" value="lannister">
+                <img class="house-img"src="assets/lannister.png" alt="lannister lion">
+                Lannister
+            </label>
+            <label>
+                <input type="radio" name="house" value="stark">
+                <img class="house-img" src="assets/stark.png" alt="stark wolf">
+                Stark
+            </label>
+            <label>
+                <input type="radio" name="house" value="targaryen">
+                <img class="house-img" src="assets/targaryen.png" alt="targaryen dragon">
+                Targaryen
+            </label>
+        </fieldset>
+        <fieldset>
+            <legend>Are you caffinated?</legend>
+            <label>
+                <input type="radio" name="caffeinated" value="yes" required>Yes
+            </label>
+            <label>
+                <input type="radio" name="caffeinated" value="no" required>No
+            </label>
+        </fieldset>
+        <button>Play the Game</button>
+        
+        </form>
+    </main>
+
+    <footer>
```

Comment:
Not sure if you plan on putting things inside this footer tag, but if not you can delete it.